lxd-qemu-snap/snapcraft.yaml

```diff
@@ -0,0 +1,213 @@
+name: qemu-for-lxd
+base: core24
+assumes:
+  - snapd2.63
+version: '1'
+grade: devel
+summary: External Qemu for LXD
+description: |-
+ External Qemu for LXD
+
+contact: lxd@lists.canonical.com
+issues: https://github.com/canonical/lxd/issues
+source-code: https://github.com/canonical/lxd
+website: https://ubuntu.com/lxd
+confinement: strict
+
+slots:
+  external-qemu:
+    interface: content
+    content: external-qemu-binaries
+    read:
+      - $SNAP/opt/external-qemu
+
+parts:
+  gpu-2404:
+    after:
+      - qemu
+    source: https://github.com/canonical/gpu-snap.git
+    plugin: dump
+    override-prime: |
+      craftctl default
+      ${CRAFT_PART_SRC}/bin/gpu-2404-cleanup mesa-2404 nvidia-2404
+    prime:
+      - bin/gpu-2404-wrapper
```

Comment:
Should the `-cleanup` script also be primed?
The [gpu-2404-cleanup](https://github.com/canonical/gpu-snap/blob/main/bin/gpu-2404-cleanup) script is a symlink to [graphics-core22-cleanup](https://github.com/canonical/gpu-snap/blob/main/bin/graphics-core22-cleanup). I don't know if the `prime` logic is smart enough to cope with that.

Reply:
I don't think that we need to prime it as we just want to use it once during snap build time. And we doing it in the line `${CRAFT_PART_SRC}/bin/gpu-2404-cleanup mesa-2404 nvidia-2404`

Reply:
OK, I hadn't realize it was a copy-n-paste from https://mir-server.io/docs/the-gpu-2404-snap-interface
Have you been able to test this? Cause when reading the `gpu-2404-cleanup` script, I think it will error out due to `nvidia-24.04` not existing in the [lists directory](https://github.com/canonical/gpu-snap/tree/main/lists).

Reply:
nope, I'm not specifically tested how this script works. I haven't seen any build time errors or warnings.

Reply:
I believe that `nvidia-2404` will be added soon and as we pull it in dynamically it will work properly.